### PR TITLE
CEDS - 1771 :  two new pages added

### PR DESCRIPTION
**Pull request by apodi**
Country of Routing Codes
Total Package Quantity

### CODE REVIEW

File: test/views/declaration/DeclarantDetailsViewSpec.scala [hunk 1]
```diff
@@ -32,7 +32,7 @@ import views.html.declaration.declarant_details
 import views.tags.ViewTest
 
 @ViewTest
-class DeclarantDetailsViewSpec extends UnitViewSpec with ExportsTestData with CommonMessages with Stubs {
+class DeclarantDetailsViewSpec extends UnitViewSpec with ExportsTestData with CommonMessages with Stubs with Injector {
```
Comment (PauloPortugal) [inline test/views/declaration/DeclarantDetailsViewSpec.scala]: Should we be testing that the link is visible in the View?

**Comment by PauloPortugal** [comment]
You also need to fix the code conflicts. Since you have started with a merge, probably you can continue with that :)

File: test/views/declaration/ConsigneeDetailsViewSpec.scala [hunk 2]
```diff
@@ -32,7 +32,7 @@ import views.html.declaration.consignee_details
 import views.tags.ViewTest
 
 @ViewTest
-class ConsigneeDetailsViewSpec extends UnitViewSpec with CommonMessages with Stubs {
+class ConsigneeDetailsViewSpec extends UnitViewSpec with CommonMessages with Stubs with Injector {
```
Comment (PatrykRudnicki) [inline test/views/declaration/ConsigneeDetailsViewSpec.scala]: Injector is not necessary here

File: test/views/declaration/DeclarantDetailsViewSpec.scala (hunk 1 above)
Comment (PatrykRudnicki) [reply test/views/declaration/DeclarantDetailsViewSpec.scala]: You're not using Injector here

File: test/views/declaration/destinationCountries/RoutingCountryQuestionViewSpec.scala [hunk 3]
```diff
@@ -26,7 +26,7 @@ import unit.tools.Stubs
 import views.declaration.spec.UnitViewSpec
 import views.html.declaration.destinationCountries.routing_country_question
 
-class RoutingCountryQuestionViewSpec extends UnitViewSpec with Stubs with ExportsTestData {
+class RoutingCountryQuestionViewSpec extends UnitViewSpec with Stubs with ExportsTestData with Injector {
```
Comment (PatrykRudnicki) [inline test/views/declaration/destinationCountries/RoutingCountryQuestionViewSpec.scala]: You're not using injector here

File: app/views/declaration/total_package_quantity.scala.html [hunk 4]
```diff
@@ -47,6 +49,9 @@
             labelClass = Some("")
         )
 
+        @components.details(messages("site.details.summary_text_this")){
+            @tariffBody
+        }
```
Comment (PatrykRudnicki) [inline app/views/declaration/total_package_quantity.scala.html]: Please add test for this component

File: app/views/declaration/destinationCountries/country_of_routing.scala.html [hunk 5]
```diff
@@ -50,7 +52,9 @@
             messages(s"declaration.${page.id}.empty"),
             AutoCompleteItem.fromCountry(Countries.allCountries, _.countryCode)
         )
-
+        @components.details(messages("site.details.summary_text_this")){
+            @tariffBody
+        }
```
Comment (PatrykRudnicki) [inline app/views/declaration/destinationCountries/country_of_routing.scala.html]: Please add test for it

File: test/views/declaration/ConsigneeDetailsViewSpec.scala (hunk 2 above)
Comment (apodi) [reply test/views/declaration/ConsigneeDetailsViewSpec.scala]: done

File: test/views/declaration/destinationCountries/RoutingCountryQuestionViewSpec.scala (hunk 3 above)
Comment (apodi) [reply test/views/declaration/destinationCountries/RoutingCountryQuestionViewSpec.scala]: done

File: app/views/declaration/destinationCountries/country_of_routing.scala.html [hunk 6]
```diff
@@ -26,6 +26,8 @@
 
 @(mode: Mode, form: Form[String], page: CountryPage)(implicit request: JourneyRequest[AnyContent], messages: Messages)
 
+@tariffBody = { @Html(messages("declaration.routingQuestion.help.bodyText", components.details_content_link())) }
```
Comment (bambuchaAdm) [inline app/views/declaration/destinationCountries/country_of_routing.scala.html]: This is single expression - can we inline this ?

**Pull request closed without merging**